raiden-dapp/src/components/account/Withdrawal.vue

```diff
@@ -156,15 +154,24 @@ export default class Withdrawal extends Mixins(BlockieMixin) {
   withdraw: Token | null = null;
 
   async mounted() {
-    this.balances = await this.$raiden.getRaidenAccountBalances();
-
-    if ((this.udcToken.balance as BigNumber).gt(Zero)) {
-      this.balances = uniqBy(
-        [...this.balances].concat(this.udcToken),
-        (token) => token.address
-      );
-    }
+    const balancesWithoutMainnetUtilityToken = await this.$raiden.getRaidenAccountBalances();
+    const balancesWithMainnetUtilityToken = uniqBy(
+      [...balancesWithoutMainnetUtilityToken].concat(this.udcToken),
+      (token) => token.address
+    );
+    const updatedBalances = await Promise.all(
+      balancesWithMainnetUtilityToken.map(async (token) => ({
+        ...token,
+        balance: await this.$raiden.raiden.getTokenBalance(
```

Comment:
Accessing `$raiden.raiden` isn't a good practice here: since we have `RaidenService`, it should wrap all accesses to the `Raiden` SDK instance. You may expose it, or the function to get all tokens balances, maybe with a flag for either main account or raiden account, and use it from there.

Reply:
Yes, I changed that already. :)